package.json

```diff
@@ -5,13 +5,15 @@
   "description": "A basic Plasmo extension.",
   "author": "dlaun",
   "packageManager": "npm@8.19.2",
+  "type": "module",
```

Comment:
This can't be committed as it breaks the extension from working, just has to be manually added and removed for now if you need to test with ts-node-esm